code/datums/ammo/bullet/tank.dm

```diff
@@ -4,9 +4,14 @@
 //======
 */
 
+//To do: Move the rest of the Tank ammo into here so its all organized. They are currently all over the place.
+
+//Autocannon Ammo//
+
 /datum/ammo/bullet/tank/flak
 	name = "flak autocannon bullet"
 	icon_state = "autocannon"
+	sound_hit  = list('sound/weapons/sting_boom_small1.ogg')
```

Comment:
add the path to file in get_sfx and link it here instead, look at how other sound_hit is done, this is mostly for consistency though.